### ISSUE TITLE
Multi language needed + Dark mode in Mobile explorer

### ISSUE THREAD

**Issue by Speedevs** [matter-labs/block-explorer#178]
Looks like someone copied my work 🤣🤣 of what I wrote

https://github.com/matter-labs/zksync-web-era-docs/issues/916

Which is actually here.

More multi language needed in the explorer

https://explorer.zksync.io/
![Screenshot_20240218_163428_Chrome](https://github.com/matter-labs/block-explorer/assets/45816964/ee0a0a8d-a33c-4ef5-819c-eff29dad53e9)
 (like CN)

_more languages can be added_

**Don't copy what you don't understand lmao**

**Comment by Speedevs** [comment]
![Screenshot_20240218_163939_Chrome](https://github.com/matter-labs/block-explorer/assets/45816964/fdcd9d0d-59c0-408f-8639-2100eb1b2a73)

Directly to be specific in what I meant

**Comment by kelreel** [comment]
Hey, https://github.com/matter-labs/block-explorer/pull/187 MR with RU lang. There is also a PR with Turkish, I think even more languages will be available in the future

**Comment by Speedevs** [comment]
> Hey, #187 MR with RU lang. There is also a PR with Turkish, I think even more languages will be available in the future

Also the dark mode request too

**Comment by Speedevs** [comment]
> Hey, #187 MR with RU lang. There is also a PR with Turkish, I think even more languages will be available in the future

Yeah , hopefully it will be able and easier for people to set there various languages of there choice , because could not find the language I wanted some people to use lol
